src/components/badges/BadgeWrapper.tsx

```diff
@@ -20,18 +21,24 @@ const badgeWrapper = (minted: boolean) =>
     alignItems('items-center'),
     borderRadius('rounded-lg'),
     backgroundColor(minted ? 'bg-primary-dimmed' : 'bg-primary-background'),
-    padding('px-4', 'py-4')
+    padding('px-4', 'py-4'),
+    gridColumn('col-span-1', 'lg:col-span-2')
   )
 
 export default function ({
   children,
   minted,
 }: ChildrenProp & { minted: boolean }) {
+  const lastWideBadgeSmToLg = 'smToLg:odd:last:col-span-full'
+
   return (
     <div
       className={classNamesToString(
         badgeWrapper(minted),
-        'last:odd:col-span-full'
+        'lg:each-new-row-in-3-cols:last:col-span-full',
+        'lg:each-new-row-in-3-cols:pre-last:col-span-3',
+        'lg:each-2nd-element-in-3-cols:last:col-span-3',
+        lastWideBadgeSmToLg
       )}
```

Comment:
Why won't you place the whole `classNamesToString` into the const variable?
```
const badgeWrapper = classNamesToString(
        badgeWrapper(minted),
        'lg:each-new-row-in-3-cols:last:col-span-full',
        'lg:each-new-row-in-3-cols:pre-last:col-span-3',
        'lg:each-2nd-element-in-3-cols:last:col-span-3',
        'smToLg:odd:last:col-span-full'
      )
```

Reply:
Oh, okay, I didn't understand you